pkg/kn/commands/service/configuration_edit_flags.go

```diff
@@ -434,6 +438,17 @@ func (p *ConfigurationEditFlags) Apply(
 		}
 	}
 
+	if cmd.Flags().Changed("scale") {
+		err = servinglib.UpdateMaxScale(template, p.Scale)
```

Comment:
What is if `--max-scale` is also given on the command line ? Which one takes precedence ? Or should an error be thrown ? 

IMO we should return an error if both `--scale` and either `--min-scale` or `--max-scale` is provied.

Reply:
I need to add this functionality and also tests for it.